draft-ietf-quic-recovery.md

```diff
@@ -1269,19 +1269,39 @@ timers wake up late. Timers set in the past fire immediately.
 Pseudocode for SetLossDetectionTimer follows:
 
 ~~~
-GetEarliestTimeAndSpace(times):
-  time = times[Initial]
+GetLossTimeAndSpace():
+  time = loss_time[Initial]
   space = Initial
   for pn_space in [ Handshake, ApplicationData ]:
-    if (times[pn_space] != 0 &&
-        (time == 0 || times[pn_space] < time) &&
-        # Skip ApplicationData until handshake completion.
-        (pn_space != ApplicationData ||
-          IsHandshakeComplete()):
-      time = times[pn_space];
+    if (time == 0 || loss_time[pn_space] < time):
+      time = loss_time[pn_space];
       space = pn_space
   return time, space
 
+GetPtoTimeAndSpace():
+  duration = smoothed_rtt +
+      max(4 * rttvar, kGranularity) * (2 ^ pto_count)
+  # Arm PTO from now when there are no inflight packets
+  if (no in-flight packets):
+    return (now() + duration), Initial
```

Comment:
This breaks the following case: As a client, I need to send a Handshake packet to unblock the server's amplification limit, if the Initial packet number space was already dropped.

Reply:
Thanks.  The amplification factor is the gift that keeps on giving.